packages/ra-ui-materialui/src/input/AutocompleteInput.tsx

```diff
@@ -342,6 +368,12 @@ If you provided a React element for the optionText prop, you must also provide t
                 return option?.name;
             }
 
+            if (!isListItem && option[optionValue || 'id'] === emptyValue) {
```

Comment:
This change prevents the empty choice from getting processed by `getRecordRepresentation` when `isFromReference` is `true`